### ISSUE TITLE
question for fusion_net

### ISSUE THREAD

**Issue by ChangjianLi** [zhangy76/PhysMoP#4]
Hi, great work. I have a little question about the calculation about weight_t ---> weight_t = torch.tanh(self.fusion_net(motion_pred_data[:, config.hist_length:].clone().detach(), motion_pred_physics_pred[:, config.hist_length:].clone().detach(), motion_feats.clone().detach(), time_idx.clone().detach())) ** 2 which is calculated using detach() parameters, I wondering why

**Comment by zhangy76** [comment]
Our framework basically consists of three components, the data-drive, the physics, and the fusion model. Adding ".detach()" is intended to train each model individually.

**Comment by zhangy76** [comment]
Close due to being inactive, please feel free to reopen.
